packages/task/src/node/task-server.slow-spec.ts

```diff
@@ -5,29 +5,24 @@
  * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-import * as chai from 'chai';
 import 'mocha';
-import * as chaiAsPromised from 'chai-as-promised';
 import { testContainer } from './test-resources/inversify.spec-config';
 import { BackendApplication } from '@theia/core/lib/node/backend-application';
 import { TaskExitedEvent, TaskInfo, TaskServer, TaskOptions, ProcessType } from '../common/task-protocol';
 import { TaskWatcher } from '../common/task-watcher';
 import * as ws from 'ws';
 import * as http from 'http';
 import * as https from 'https';
+import * as test from '@theia/core/lib/common/test/test';
```

Comment:
Wouldn't `import { throwsAsync } from ...` work here? This way we wouldn't have to call test.throwsAsync() every time.

Reply:
In the case of using a utility module I have the habit of doing the `module.function()` so that upon reading it looks explicit. I would pick-import objects, but functions ?

In the same fashion I think it is possible to change this:
```ts
// happens a lot
import * as chai from 'chai';
const expect = chai.expect

// replacement
import { expect } from 'chai';
```

So is there a convention for the importations ?

Reply:
I don't think there is a written convention at the moment. And yes I believe we could use `import { expect } from 'chai'` afaik. But usually we would declare something like `const expect = chai.expect` right after, as to not bother with having to call `chai.expect()` every time.

We already import functions at several places in our code so I think it's fine to do it this way. To me it's just simpler as a developer and having to call `module.function` every time seems like noise.